www/js/main.js

```diff
@@ -170,19 +170,28 @@ define([
 			menu.add("createQuantityNodeButton", function(e){
 				event.stop(e);
 				//give a fake id for now
```

Comment:
You can remove this comment now. :)  

Also remove the same comment from line 185, and the dead code from line 190.